src/Middleware/EventDispatchMiddleware.php

```diff
@@ -8,6 +8,7 @@
 use Psr\Http\Message\RequestInterface;
 use Psr\Http\Message\ResponseInterface;
 use Symfony\Component\EventDispatcher\EventDispatcherInterface;
+use Symfony\Contracts\EventDispatcher\Event;
```

Comment:
replace this import with: `use EightPoints\Bundle\GuzzleBundle\Events\Event;` and tests will pass :slightly_smiling_face:

Reply:
Done. Builds are passing :tada: